### ISSUE TITLE
Include -> save() as part of chain

### ISSUE THREAD

**Issue by turbo124** [beganovich/snappdf#1]
it would be helpful to have the file save as part of the pdf call..

ie instead of

```php
$pdf = $snappdf
    ->setHtml('<h1>Hello world!</h1>')
    ->generate();

file_put_contents('my-awesome.pdf', $pdf);
```

this

```php

$pdf = $snappdf
    ->setHtml('<h1>Hello world!</h1>')
    ->save($path);
```

**Comment by beganovich** [comment]
Hey!

Thanks for the suggestion. I had in the first place, however, it was required to use that to even generate PDF. Then I switched to this because it will help save the content of PDF to different disks.

Any idea how to cover, for example, saving PDF on S3 without saving to local and then moving to a different disk?

Example:

```php
$pdf = $snappdf->generate();

Storage::disk('s3')->put('example.pdf', $pdf);
```

**Comment by turbo124** [comment]
so i think 80% of users will use the configured disk, others would need to just manually generate() and then save to the disk they want... so the above functionality would be in addition to the ->generate() method

**Comment by beganovich** [comment]
All right, thanks for the awesome suggestion. 🎉

**Comment by beganovich** [comment]
Hey there!

Thanks for the suggestion, this has been implemented in the v1.3.0 release:

https://github.com/beganovich/snappdf/releases/tag/v1.3.0

Thanks!
